astropy/modeling/core.py

```diff
@@ -3262,6 +3262,14 @@ def submodel_names(self):
                 newnames.append(item)
         return tuple(newnames)
 
+    @property
+    def submodel_types(self):
```

Comment:
I think "types" is misleading. This returns the immediate class names, not some types (which might be confused with numpy data types and so on). Are you open to renaming the property?

For example, if I use the following, I would see "Bozo" in the output, right? Not really a type.

```
class Bozo(Gaussian1D):
   pass
```

Reply:
How about `submodel_classnames`?

Reply:
Fine by me as along as @WilliamJamieson and/or @perrygreenfield okay with that. Thanks!